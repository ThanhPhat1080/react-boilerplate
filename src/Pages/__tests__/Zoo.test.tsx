// Libraries
import * as renderer from 'react-test-renderer';

// Components
import { BrowserRouter } from 'react-router-dom';
import { Zoo } from '@Pages';

describe('Testing Zoo', () => {
  test('It should render correctly', () => {
    const component = renderer
      .create(
        <BrowserRouter>
          <Zoo />
        </BrowserRouter>,
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
