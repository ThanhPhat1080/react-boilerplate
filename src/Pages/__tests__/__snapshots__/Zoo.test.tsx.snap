// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Zoo It should render correctly 1`] = `
Array [
  <h1>
    Welcome to the Zoo !
  </h1>,
  <div>
    We have
     
    <span
      aria-label="list animals"
      role="img"
    >
      🐕 🦊 🐒 🦍 🦥 🦒
    </span>
  </div>,
  <nav>
    <a
      href="/"
      onClick={[Function]}
    >
      Back to HomePage
      <span
        aria-label="home"
        role="img"
      >
        🏠
      </span>
    </a>
  </nav>,
]
`;
